dockerfiles/functional-tests/docker-entrypoint.sh

```diff
@@ -56,6 +56,11 @@ if [ -d "/root/che" ]; then
   cd /root/rh-che/
 fi
 
+#next lines added to be able to gather logs for issue https://github.com/redhat-developer/che-functional-tests/issues/476
```

Comment:
I'm thinking whether it would be better to download latest version, or stick with some specific one...
Anyway I think downloading at least 3.11 shouldn't hurt  ;-)

And also... Wouldn't it be better to move this code to dockerfile? That way we wouldn't need to download oc every time we are running this container.

Reply:
Version updated, code moved to dockerfile.